src/include/network/services/dns.rb

```diff
@@ -128,22 +112,25 @@ def initialize_network_services_dns(include_target)
             Hostname.ValidDomain
           )
         },
-        "DHCP_HOSTNAME"   => {
+        "DHCP_IFACES"     => {
```

Comment:
"DHCP_IFACES" entirely hides the fact that the widget is related to the hostname.